### PR TITLE
update rule typing and params

### PR DESCRIPTION
**Pull request by stephanwampouille**
This PR updates the new rule typing under the newly named `Unit` as well as makes start and finish params required for describe activity

### CODE REVIEW

File: javascript/sdk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@theprelude/sdk",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
[the remark this answers is not in the document]
Comment (stephanwampouille) [reply javascript/sdk/package.json]: changed, I thought we would have to since Patrick published it to NPM

File: javascript/sdk/lib/types.ts [hunk 2]
```diff
@@ -200,16 +200,16 @@ export interface RuleUsage {
   failed: number;
 }
 
-export interface RuleInfo {
+export interface Unit {
   rule: Rule | null;
   usage?: RuleUsage;
```
Comment (mireaulf) [inline javascript/sdk/lib/types.ts:205]: RuleUsage -> UnitUsage

File: javascript/sdk/lib/types.ts [hunk 3]
```diff
@@ -194,15 +194,15 @@ export interface Rule {
...
 
-export interface RuleUsage {
+export interface UnitUsage {
   tests: string[];
   count: number;
   failed: number;
 }
 
 export interface Unit {
   rule: Rule | null;
```
Comment (mireaulf) [inline javascript/sdk/lib/types.ts:204]: Rule :)

Comment (stephanwampouille) [reply javascript/sdk/lib/types.ts:204]: wait but I just made the change above like you said RuleUsage -> UnitUsage

**Comment by mireaulf** [comment]
This needs backend changes to return units instead of rules

**Pull request merged**